docs/geedocs/answer/3499938.html

```diff
@@ -62,18 +66,17 @@
 <li>SUSE Linux Enterprise Server 10</li>
```

Comment:
Since this is for an old version, I'm assuming Google did the testing before they released it.